rules/windows/process_creation/proc_creation_win_wmic_system_info_discovery.yml

```
@@ -0,0 +1,40 @@
+title: System Information Discovery with WMIC
+id: 9d5a1274-922a-49d0-87f3-8c653483b909
+status: experimental
+description: Detect use of the WMI command-line (WMIC) utility to identify and display various system information, including OS, CPU, GPU, and disk drive names; memory capacity; display resolution; and baseboard, BIOS, and GPU driver products/versions. Some of these commands were used by Aurora Stealer in late 2022/early 2023.
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/a2ccd19c37d0278b4ffa8583add3cf52060a5418/atomics/T1082/T1082.md#atomic-test-25---system-information-discovery-with-wmic
+    - https://nwgat.ninja/getting-system-information-with-wmic-on-windows/
+    - https://blog.sekoia.io/aurora-a-rising-stealer-flying-under-the-radar
+    - https://blog.cyble.com/2023/01/18/aurora-a-stealer-using-shapeshifting-tactics/
+    - https://app.any.run/tasks/a6aa0057-82ec-451f-8f99-55650ca537da/
+author: TropChaud
+date: 2023/01/26
+tags:
+    - attack.discovery
+    - attack.t1082
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_wmic:
+        - Description: 'WMI Commandline Utility'
+        - OriginalFileName: 'wmic.exe'
+        - Image|endswith: '\WMIC.exe'
+        - CommandLine|contains: 'wmic'
```

```suggestion
        - CommandLine|contains:
            - 'wmic '
            - 'wmic.exe'
```
wmic is a short string which might appear in other contexts in the command line

---

I would remove the CLI as a whole as it's covered by the Image and OFN